### DISCUSSION TITLE
Excessively restrictive reviews when applying for access to restricted roles as a public developer

### DISCUSSION THREAD

**Discussion by jlevers** [amzn/selling-partner-api-models#588] [Ideas]
Hi SP API team,

Thanks for all your work on this API – it's a big improvement over MWS to have a JSON-based system, and notifications in particular are really, really helpful.

I feel like a lot of the work you've done to make a superior replacement for MWS is being defeated by the incredibly restrictive, vague, and difficult application process for getting access to the SP API, particularly for getting PII access as a public developer. I understand that you want to protect Amazon customers, and that's a valuable goal, but customers are also getting hurt by the anti-competitive nature of the approval process.

I work with many clients who are established e-commerce service providers on other platforms who are trying to get access to the SP API in order to provide various Amazon-related services to their customers (they often have 10s or 100s of thousands of customers). Over and over, they are getting rejected on the basis of not being sufficiently differentiated from the existing offerings in their category. I think that this is a bizarre requirement for approval. Why should the Amazon application reviewers be the arbitrators of whether or not a new business is viable? They should be allowed to compete in the Marketplace App Store with everyone else.

Often, the businesses who are being denied access to PII are businesses that _already have_ PII access via MWS, and are just trying to port their system over to the SP API. They will go under if they can't get access to PII via the SP API, since MWS is being deprecated. I fail to see the benefit to customers of destroying mid-sized businesses by denying them access to data that, on any other e-commerce platform, is *so* much easier to access.

I think customers would benefit greatly if the businesses helping to serve them were allowed to spend their time developing better seller tools, instead of spending months trying to get access to the basic data that they need to get started.

I've been deeply involved in it from the beginning, maintain one of [the most heavily used SDKs](https://github.com/jlevers/selling-partner-api) for it, and have built countless systems that make use of it. This is an observation based on a LOT of time spent dealing with SP API applications, across many businesses.

All responses are appreciated, and I want to reiterate, I understand that the team is doing their best with the instructions they've been given – but on this topic, I think change is necessary.

**Comment by rugved1991** [comment]
Hi @jlevers,

Thanks for your feedback! This is really insightful. 
I will share this feedback with our internal teams. We appreciate all the work you are doing to build with the new SP APIs. 

Best,
Rugved
Solutions Architect, SP API

**Reply by jlevers** [reply]
Hi @rugved1991,

Appreciate your response. If it would be possible to have a direct discussion with the team doing PII approvals, that would be incredibly helpful. I currently have multiple clients with thousands or tens of thousands of customers who can't get their customers onto Amazon (but support _many_ other marketplaces), which is bad for those potential sellers, bad for my clients, and bad for Amazon as a marketplace.

Is there any chance of discussing this issue with the team? My email address is <first name> [at] highsidelabs.co.

Thank you,
Jesse

**Reply by jlevers** [reply]
Also, I just want to be extra clear: from what I can see, there are many other discussions (#2938, https://github.com/amzn/selling-partner-api-docs/discussions/2205#discussioncomment-2594015, the comments in amzn/selling-partner-api-docs#2712, amzn/selling-partner-api-docs#2598, amzn/selling-partner-api-docs#2819, etc) about this particular problem popping up here. Some people who had fully functional systems using MWS are unable to port them over to the SP API because they are getting rejected for PII access with generic canned responses, and then told the rejection is final when they try to contest them.

I understand that Amazon is trying to protect sensitive customer information, but there is no other major ecommerce platform that makes it this difficult to build something as simple as a custom shipping system.

**Reply by jlevers** [reply]
@rugved1991 checking in about the above again. A real discussion of this with the people who decide how to structure the approval process would be deeply appreciated.
